pkg/logs/tail/tail.go

```diff
@@ -0,0 +1,28 @@
+package tail
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/aybabtme/humanlog"
+)
+
+// TailLogs lets us run `lazygit --logs` to print the logs produced by other lazygit processes.
+// This makes for easier debugging.
+func TailLogs(logFilePath string) {
```

Comment:
moved this into its own package to ensure we have as few dependencies as possible in the logs package